docs/cloud-integration.md

```diff
@@ -276,7 +276,7 @@ under-reported with Hadoop versions before 3.3.1.
 Here is the documentation on the standard connectors both from Apache and the cloud providers.
 
 * [OpenStack Swift](https://hadoop.apache.org/docs/current/hadoop-openstack/index.html).
-* [Azure Blob Storage and Azure Datalake Gen 2](https://hadoop.apache.org/docs/current/hadoop-aws/tools/hadoop-aws/index.html).
+* [Azure Blob Storage and Azure Datalake Gen 2](https://hadoop.apache.org/docs/current/hadoop-azure/abfs.html).
```

Comment:
The link is specifically for ABFS actually. Should we provide the link for `Azure Blob Storage`: https://hadoop.apache.org/docs/current/hadoop-azure/index.html ?

Reply:
Thanks for the feedback! Yes, the link I provided is for ABFS & Data Lake Gen 2 specifically. Which option would you prefer:

1. Change the link text to say "Azure Blob Storage" and point to https://hadoop.apache.org/docs/current/hadoop-azure/index.html 
2. Change the link text to say "Azure Blob Filesystem and Azure Data Lake Gen 2" and point to https://hadoop.apache.org/docs/current/hadoop-azure/abfs.html
3. Have both links mentioned above available.

Let me know which one you prefer and I'll make a change.

Reply:
I would prefer third one - both links mentioned separately.

Reply:
Updated.